src/discussions/common/AlertBanner.jsx

```diff
@@ -20,24 +22,52 @@ function AlertBanner({
   const isQuestion = postType === ThreadType.QUESTION;
   const classes = isQuestion ? 'bg-success-500 text-white' : 'bg-dark-500 text-white';
   const iconClass = isQuestion ? CheckCircle : Verified;
+  const endorsedByLabels = { Staff: 'Staff', 'Community TA': 'TA' };
   return (
     <>
       {content.endorsed && (
-        <Alert variant="plain" className={`p-3 m-0 rounded-0 shadow-none ${classes}`} icon={iconClass}>
+        <Alert
+          variant="plain"
+          className={`p-3 m-0 shadow-none ${classes}`}
+          style={{ borderRadius: '0.375rem 0.375rem 0 0' }}
+          icon={iconClass}
+        >
           <div className="d-flex justify-content-between">
-            <strong>{intl.formatMessage(
+            <strong className="lead">{intl.formatMessage(
               isQuestion
                 ? messages.answer
                 : messages.endorsed,
             )}
             </strong>
-            <span>
+            <span className="d-flex align-items-center">
               {intl.formatMessage(
                 isQuestion
                   ? messages.answeredLabel
                   : messages.endorsedLabel,
-              )}&nbsp;
-              <Hyperlink>{content.endorsedBy}</Hyperlink>&nbsp;
+              )}
+              <span className="mx-2">{content.endorsedBy}</span>
+
+              {content.endorsedByLabel === 'Staff' ? (
+                <Icon
+                  style={{
+                    width: '1rem',
+                    height: '1rem',
+                  }}
+                  src={Institution}
+                />
```

Comment:
can we add a test case for this condition as well? it will remove the codecov failure.

Reply:
Sure